### DISCUSSION TITLE
Error when i try to load a glsl shader

### DISCUSSION THREAD

**Discussion by Shiv0r** [bnpr/Malt#275] [Q&A]
```
D:/User/Shiv0r/Desktop/Shader/basic.mesh.glsl:3: error: Can't open include file "Pipelines/NPR_Pipeline.glsl"

    #include "Pipelines/NPR_Pipeline.glsl"

    from C:/Users/shiv0/AppData/Local/Temp/tmpg3f1uiwe: 1:    #include "basic.mesh.glsl"

1 error in preprocessor.


```
Im using right now Blender 3.0 and a older version of the Beer/Malt (1.0.0beta). But i have the same issue with the latest Version and 3.1

Im also using the shader examples from the first release.

The Error looks like that in the Lookdev View:

![image](https://user-images.githubusercontent.com/35176958/159001039-9a38e950-3575-477e-a86f-3292315ab10a.png)

**Comment by pragma37** [comment]
> Im also using the shader examples from the first release.

That's the issue. The shader API was redesigned for 1.0 and older shaders are not compatible.
That's why the shader examples are not present in newer releases.

For newer shader examples check #94 .
